packages/mds-collector-backend/service/provider.ts

```diff
@@ -77,7 +77,8 @@ export const CollectorServiceProvider: ServiceProvider<CollectorService> & Proce
 
   registerMessageSchema: async (schema_id, schema) => {
     try {
-      const validator = SchemaValidator(schema)
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const validator = SchemaValidator(schema as any)
```

Comment:
What's this about?

Reply:
This is about having a schema validation service that _cannot_ be validated or compiled up front, but rather runs on a dynamic definition supplied by the caller. It is 100% _**any**_